src/graphql/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@rainbow-me/graphql-client",
+  "private": true,
+  "scripts": {
+    "codegen": "graphql-codegen --config codegen.config.js"
+  },
+  "devDependencies": {
```

Comment:
Did you see that there is a react-query module that generates the react-query query and mutations automatically using this tool

Reply:
Yeah I noticed that! Pretty cool, but I decided not to use it for the following reasons:

1. Not all of our async state are GraphQL requests, so having 2 approaches (RQ graphql-codegen plugin & RQ RFC) might be a bit weird.
2. Tweaking the RQ graphql-codegen plugin to match our RFC (to include prefetchers, custom types, etc) might be more effort than what it is worth.

Reply:
Yeah that’s a very fair point!